Tests/conf.json

```diff
@@ -20,6 +20,31 @@
     "testTimeout": 160,
     "testInterval": 20,
     "tests": [
+        {
+            "playbookID": "T1036 - Masquerading - Test",
+            "integrations": "VirusTotal (API v3)",
+            "instance_names": "virus_total_v3"
+        },
+	    {
+            "playbookID": "T1059 - Command and Scripting Interpreter - Test",
+            "integrations": "LOLBAS Feed"
```

Comment:
Check with Daria if this will be turned on automatically when it is installed

Reply:
Daria said it should work properly.